src/media-query-provider.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import shallowequal from 'shallowequal';
+import _ from 'lodash';
```

Comment:
this will add the whole of lodash to the bundle 
Best to import `debounce` like: `import debounce from 'lodash/debounce';`

Reply:
You’re right. Will do so

Reply:
Removed and replaced with a simple debounce function